test/ConfigParsing/dummy.js

```diff
@@ -0,0 +1,6 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+(function() {})();

```

Comment:
Does this test require a no-op function? Copyright check is fine with completely empty files with no copyright, so if you just need a dummy.js file you can have it as a completely empty file.